_posts/2022-08-08-Jeffy-Jeffy.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: "Inside the Mind of a Computational Chemist: Emmanual Cortes" 
+author: Jefy Jeffy 
+--- 
+
+Today, I had a chance to meet (virtually) and speak with a real life computational chemist, Emmanuel Cortes. Cortes was born in Mexico but movedd to the United States at the 4 and has lived in California ever since. Currently, he lives in the Bay Area and loves the weather. He has not had a chance to go back to Mexico since coming to States, but hopes he has a chance to visit soon. Emmanuel mentioned that one of his favorite foods is Lasagna, and pastas in general; however, he dislikes mushrooms due to their texture. I claim that he would absolutely love mushroom, if prepared well. So here's to hoping that Emmanuel dares to eat mushrooms cooked in the south Indian style. 
```

Comment:
I think you meant "at the age of 4".

Reply:
yep thats what I meant. I can go ahead and fix it and recommit.